android/README.md

```diff
@@ -22,35 +22,61 @@ CronetNetworking.initializeCronetEngine(getApplicationContext(), "YOUR-ENVOY-URL
 ```
 ## Envoy url format
 
-Http/Https:
- - http://domain:port (no trailing slash)
- - https://domain:port (no trailing slash)
+### HTTP/HTTPS
 
-Shadowsocks:
- - ss://encrypted login@ip:port/
+Envoy uses a nonstandard HTTP/HTTPS proxy where the origial request is passed in a header. Unencrypted HTTP is supported, but not recommeded. Cronet currently does not support proxying over QUIC / HTTP/3, so HTTP 1.1 or 2 is required. Server side setup is documented [here](./native/README.md) and a working example can be found [here](https://gitlab.com/stevenmcdonald/envoy-proxy-examples/-/tree/main/http_proxy).
```

Comment:
./native/... produces this url:

https://github.com/greatfire/envoy/blob/docs-update/android/native/README.md

i think you want ../native/